internal/publish/database/exposure.go

```diff
@@ -380,6 +401,15 @@ type InsertAndReviseExposuresRequest struct {
 	// exposures are included that are not part of the revision token. This exists
 	// to support roaming scenarios. This is only used if RequireToken is true.
 	AllowPartialRevisions bool
+
+	// The following operations are for federation.
+
+	// If true, if a key is determined to be a revsion, it is skipped.
+	SkipRevions bool
+	// If true, only revisions will be processed.
+	OnlyRevisions bool
```

Comment:
I think we need validation here to make sure someone doesn't set SkipRevisions and OnlyRevisions. Or make this a `RevisionMode` field.

Reply:
added